rust/android/Cargo.toml

```diff
@@ -4,13 +4,11 @@ version = "1.0.1"
 authors = ["Aditya Kulkarni <aditya@zecwallet.co>", "Zingolabs <zingo@zingolabs.com>"]
 edition = "2021"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
 [dependencies]
-jni = { version = "0.10.2", default-features = false }
-android_logger = "0.8.6"
-log = "0.4.8"
+jni = { version = "0.19", default-features = false }
+android_logger = "0.11"
+log = "0.4"
```

Comment:
I don't see any problem with this, but I don't know why the version seems decremented. @zancas do you know?

Reply:
Also, this same dependency is specified in `/rust/lib/Cargo.toml` with `log = "0.4.8"`

Reply:
I looked it up, and `"^4.8.0"` is exactly equivalent to `"4.8.0"`, but shouldn't stop at the next version whole number, it should just go to the highest compatible version.

Reply:
OK..   I think whatever works.   This shouldn't be blocking for this PR.